packages/react-devtools-shell/package.json

```diff
@@ -3,9 +3,7 @@
   "name": "react-devtools-shell",
   "version": "0.0.0",
   "scripts": {
-    "start": "yarn start:app",
-    "start:app": "cross-env NODE_ENV=development cross-env TARGET=local webpack-dev-server --open-page app.html",
-    "start:multi": "cross-env NODE_ENV=development cross-env TARGET=local webpack-dev-server --open-page multi.html"
+    "start": "cross-env NODE_ENV=development cross-env TARGET=local webpack-dev-server"
```

Comment:
I renamed the `app.html` file to `index.html` and added a small nav bar up top so we can get rid of these separate targets.